<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_info</class>
 <widget class="QDialog" name="user_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="list">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>511</width>
     <height>401</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>4</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>5</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="change_name">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>200</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="change_name_Btn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>200</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>수정</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="change_vip">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>250</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="change_vi_Btn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>250</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>등급 선택</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="slt_id">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>150</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>slt_id</tabstop>
  <tabstop>change_name</tabstop>
  <tabstop>change_vip</tabstop>
  <tabstop>change_name_Btn</tabstop>
  <tabstop>change_vi_Btn</tabstop>
  <tabstop>list</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
